CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+# [v0.11.0-beta.3](https://github.com/kubermatic/kubeone/releases/tag/v0.11.0-beta.0) - 2019-12-20
+
+## Attention Needed
+
+* Kubernetes 1.14 clusters are not supported as of this release because 1.14 isn't supported by the upstream anymore
+  * It remains possible to upgrade 1.14 clusters to 1.15 and is strongly advised
```

Comment:
Sounds IMHO better as `It remains possible and is advisable to upgrade ...`.

Reply:
Done